cmake/modules/autopas_gtest.cmake

```diff
@@ -1,27 +1,27 @@
-message(STATUS "gtest - using bundled version")
-find_package(Threads REQUIRED)
-
-# Enable FetchContent CMake module
-include(FetchContent)
-
-# Build GoogleTest and make the cmake targets available
-FetchContent_Declare(
-    autopas_gtest
-    URL ${AUTOPAS_SOURCE_DIR}/libs/googletest-1.10.0.zip
-    URL_HASH MD5=82358affdd7ab94854c8ee73a180fc53
-)
-
-option(INSTALL_GTEST "" OFF)
-
-# hide options from ccmake
-mark_as_advanced(BUILD_GMOCK INSTALL_GTEST)
-
-# Prevent overriding the parent project's compiler/linker settings on Windows.
-# => Compiles gtest with correct mt(d)/md(d)
-set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
-
-FetchContent_GetProperties(autopas_gtest)
-if (NOT autopas_gtest_POPULATED)
-    FetchContent_Populate(autopas_gtest)
-    add_subdirectory(${autopas_gtest_SOURCE_DIR} ${autopas_gtest_BINARY_DIR} EXCLUDE_FROM_ALL)
-endif ()
+message(STATUS "gtest - using bundled version")
+find_package(Threads REQUIRED)
+
+# Enable FetchContent CMake module
+include(FetchContent)
+
+# Build GoogleTest and make the cmake targets available
+FetchContent_Declare(
+    autopas_gtest
+    URL ${AUTOPAS_SOURCE_DIR}/libs/googletest-1.11.0.zip
+    URL_HASH MD5=52943a59cefce0ae0491d4d2412c120b

```

Comment:
cool. Any reason for the upgrade?

Reply:
* additional internal tests that check the correctness of definition of tests.
* no more stupid warnings when using cmake >= 3.19 (1.10 allowed the usage of deprecated cmake versions, i.e. < 2.8.12)